src/clojure_for_js_devs/handlers.clj

```diff
@@ -8,3 +8,9 @@
 (defn ping-handler [redis-component]
   (println "Handling ping request")
   (redis/ping redis-component))
+
+(defn counter-handler [req redis-component]
+  (let [ip (-> req :remote-addr)
```

Comment:
I. don't think we need to do the threading here. Can simplify to `(:remote-addr req)`